### PR TITLE
feat: add topic separator option

### PR DESCRIPTION
**Pull request by RasPhilCo**

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -298,7 +301,8 @@ export class Config implements IConfig {
   }
 
   get commandIDs() {
-    return uniq(this.commands.map(c => c.id))
+    const ids = Lodash.flattenDeep(this.commands.map(c => [c.id, c.aliases]))
```
Comment (RasPhilCo) [inline src/config/config.ts:304]: @amphro let me know your thoughts, but here I'm saying `commandIDs` should return alias ids as well

Comment (RasPhilCo) [reply src/config/config.ts:304]: maybe some downstream affects I'm not thinking through

**Review by amphro — APPROVED** [review]
Nice! A few more tests around finding commands might be nice.

File: src/help/util.ts [hunk 2]
```diff
@@ -39,3 +39,32 @@ export function template(context: any): (t: string) => string {
...
+
+function collateSpacedCmdIDFromArgs(argv: string[], config: IConfig): string[] {
+  if (argv.length === 1) return argv
+
+  const ids = config.commandIDs.concat(config.topics.map(t => t.name))
+
+  const findId = (id: string, next: string[]): string | undefined => {
+    const idPresnet = (id: string) => ids.includes(id)
+    if (idPresnet(id) && !idPresnet(`${id}:${next[0]}`)) return id
+    if (next.length === 0 || next[0] === '--') return
```
[the remark this answers is not in the document]
Comment (RasPhilCo) [reply src/help/util.ts:51]: This is returning no match b/c the line would return the if was the last valid id found. This line is saying, you didn't find anything to return (line above would of returned) and you have no further argv to explore.

**Pull request merged**